core-java-modules/core-java-io-5/src/test/java/com/baeldung/outputstreamtobytearray/OutputStreamtoByteArrayUnitTest.java

```diff
@@ -0,0 +1,56 @@
+package com.baeldung.outputstreamtobytearray;
+
+import org.apache.commons.io.FileUtils;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.io.*;
+
+public class OutputStreamtoByteArrayUnitTest {
+
+    @Test
+    public void givenFileOutputStream_whenUsingFileUtilsToReadTheFile_thenReturnByteArray() throws IOException {
+        String data = "Welcome to Baeldung!";
+        String filePath = "file.txt";

```

Comment:
Let's use https://www.baeldung.com/junit-5-temporary-directory so that the file is cleaned up after the test run

Reply:
Done